res/bnd/annotations-api.jar.tmp.bnd

```diff
@@ -35,9 +35,9 @@ Provide-Capability: \
     Specification-Version=1.3
 
 -jpms-module-info: \
-    jakarta.annotation;\
+    javax.annotation.api;\
         access=32;\
         version=${annotation.spec.version}
 -jpms-module-info-options: \
-    jakarta.annotation;\
+    javax.annotation.api;\
```

Comment:
The Java EE 8 JAR uses `java.annotation` as the module name

Reply:
fixed